docs/sts/web-identity.md

```diff
@@ -24,6 +32,14 @@ There are situations when identity provider does not provide user claims in `id_
 | *Type*     | *String* |
 | *Required* | *No*     |
 
+### RoleARN
+The role ARN to use. This must be specified if and only if the web identity provider is configured with a role policy.
+
+| Params     | Value    |
+| :--        | :--      |
+| *Type*     | *String* |
+| *Required* | *No*     |
```

Comment:
Perhaps we can mention that they should read about "RoleArn" before deciding not to present it or configure it.

Reply:
Our implementation is pretty different from AWS - what should users read? In AWS they must use role ARN, but we have an alternative way (claim name) that is not available in AWS.

Reply:
We should ideally cover whatever we need to say in this doc - what do you think needs to be added?

Reply:
I was thinking to recommend this approach over claim based,  even if it's optional. Perhaps that will trigger adopting this style instead of custom claims.